modules/virt-hot-plugging-a-virtual-disk.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * virtual_machines/virtual_disks/virt-hot-plugging-virtual-disks.adoc
+
+[id="virt-hot-plugging-a-virtual-disk_{context}"]
+= Hot-plugging a virtual disk
+
+Hot-plug virtual disks that you want to add to or remove from a virtual machine instance while a virtual machine is running.
+
+.Prerequisites
+* You must have a running virtual machine to hot-plug a virtual disk.
+* You must have at least one data volume or persistent volume claims (PVC) available for hot-plugging.
+
+To hot-plug a virtual disk using the CLI, use the `addvolume` command with `virtctl`.
+
+.Procedure
+
+. Run the following command:
+
+----
+$ virtctl addvolume <virtual-machine|virtual-machine-instance> --volume-name=<datavolume-or-PVC-to-hot-plug> [--persist] [--serial=<label-name>]
+----
+
+* The `addvolume` command creates an attachment pod to attach the hot-plugged disk.
+* The optional `--persist` flag adds the hot-plugged disk to the virtual machine specification as a permanently mounted virtual disk. After running the command with this option, the virtual disk is no longer able to be hot-unplugged. In other words, the virtual disk can no longer be added to or removed from the virtual machine while the virtual machine is running. The `--persist` flag applies to virtual machines, not virtual machine interfaces.
```

Comment:
A small but critical detail with the --persist flag. The --persist flag causes the VM spec to get updated, but as long as the VM is running, you can still unplug the disk. However after you stop and start the VM, then the VM is recreated from the VM spec, and only then will you not be able to unplug the disk anymore. But as long as you don't stop and start it (or reboot, which is the same) you can still unplug it.

Reply:
Thanks for the clarification. Reworded as follows with your latest update as a Note:

_* The optional `--persist` flag adds the hot-plugged disk to the virtual machine specification as a permanently mounted virtual disk. After running the command with this option, the virtual disk is no longer able to be hot-plugged or hot-unplugged. In other words, the virtual disk can no longer be added to or removed from the virtual machine while the virtual machine is running. The `--persist` flag applies to virtual machines, not virtual machine interfaces.
+
[NOTE]
----
Specifying the `--persist` flag adds the hot-plugged virtual disk to the virtual machine specification as a permanently mounted virtual disk, but this update does not take effect until the virtual machine has been stopped, restarted, or rebooted. Until then, you can hot-plug or hot-unplug the virtual disk. 
----_